merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/SimulationDriver.java

```diff
@@ -43,10 +37,16 @@ SimulationResults simulate(
       }
 
       // Schedule the control task.
```

Comment:
Probably want to update this comment. I noticed you updated the comment in `simulateTask`, but I think that comment would make more sense here instead.

Reply:
Good catch. Done!